test/index.js

```diff
@@ -23,9 +23,10 @@ describe('Index generator', () => {
   before(() => hexo.init().then(() => Post.insert([
     {source: 'foo', slug: 'foo', date: 1e8, order: 0},
     {source: 'bar', slug: 'bar', date: 1e8 + 1, order: 10},
-    {source: 'baz', slug: 'baz', date: 1e8 - 1, order: 1}
+    {source: 'baz', slug: 'baz', date: 1e8 - 1, order: 1},
+    {source: 'qux', slug: 'qux', date: 1e8 - 8, order: 8, hidden: true}
   ])).then(data => {
-    posts = Post.sort('-date');
+    posts = Post.slice(0, -1).sort('-date');
```

Comment:
This doesn't look right, why would you want to remove the last one.

Reply:
Hi! This was to filter my added test case where I labeled as `hidden`. So this post will not appear in the generated index page.

https://github.com/hexojs/hexo-generator-index/pull/125/files#diff-5bb8db779819ddef5956a5d9d5949c05ef7445237656ca37bf2f02720271440bR27

Reply:
You're right. I was wrong.